commit f20765fdfdc2c8f47b41cb08489fdad3194a8465
Author: Eric Snowberg <eric.snowberg@oracle.com>
Date:   Wed Jul 26 17:17:25 2023 -0400

    integrity: Always reference the blacklist keyring with appraisal
    
    Commit 273df864cf746 ("ima: Check against blacklisted hashes for files with
    modsig") introduced an appraise_flag option for referencing the blacklist
    keyring.  Any matching binary found on this keyring fails signature
    validation. This flag only works with module appended signatures.
    
    An important part of a PKI infrastructure is to have the ability to do
    revocation at a later time should a vulnerability be found.  Expand the
    revocation flag usage to all appraisal functions. The flag is now
    enabled by default. Setting the flag with an IMA policy has been
    deprecated. Without a revocation capability like this in place, only
    authenticity can be maintained. With this change, integrity can now be
    achieved with digital signature based IMA appraisal.
    
    Signed-off-by: Eric Snowberg <eric.snowberg@oracle.com>
    Reviewed-by: Nayna Jain <nayna@linux.ibm.com>
    Signed-off-by: Mimi Zohar <zohar@linux.ibm.com>

diff --git a/Documentation/ABI/testing/ima_policy b/Documentation/ABI/testing/ima_policy
index 49db0ff288e5..c2385183826c 100644
--- a/Documentation/ABI/testing/ima_policy
+++ b/Documentation/ABI/testing/ima_policy
@@ -57,9 +57,9 @@ Description:
 				stored in security.ima xattr. Requires
 				specifying "digest_type=verity" first.)
 
-			appraise_flag:= [check_blacklist]
-			Currently, blacklist check is only for files signed with appended
-			signature.
+			appraise_flag:= [check_blacklist] (deprecated)
+			Setting the check_blacklist flag is no longer necessary.
+			All appraisal functions set it by default.
 			digest_type:= verity
 			    Require fs-verity's file digest instead of the
 			    regular IMA file hash.